cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.5.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Need to upgrade? #7524 just bumped the version to v0.5.0

Reply:
Looks like `make controller-gen` fixes this issue, I had to update as well

Reply:
I fixed it by having `controller-gen` download to the `.cache/tools/<hostosarch>/` directory.